### PR TITLE
Update to 0.3.0, add new maintainer

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This also adds @trungleduc as a new maintainer.

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,3 +59,4 @@ extra:
     - maartenbreddels
     - JohanMabille
     - jtpio
+    - trungleduc
```
Comment (jtpio) [inline recipe/meta.yaml:62]: @trungleduc adding you here so you can merge the conda-forge PR too

**Comment by trungleduc** [comment]
Hello, I'm happy to be a maintainer of this repo.

**Pull request merged**